### ISSUE TITLE
Devices don't connect to differents servers

### ISSUE THREAD

**Issue by KellyDi0te** [Ylianst/MeshCentral#5230]
**Describe your issue**
I'm trying to do a MeshCentral cluster with 3 differents servers. 
The replicaset is done correctly because when i change the name in server1, it also change the name on server2 and server3. 
But the devices seems to connect only to server1, making them appear offline in server2 and server3. 

Am I missing something else that I need to do ? 

**Server Software (please complete the following information):**
 - OS: Windows 10 22H2 
 - Virtualization: VirtualBox
 - Network: LAN 
 - Version: 
 - Node: 

**Client Device (please complete the following information):**
 - Device: Every type of device 
 - OS: Windows 10 22h2
 - Network: LAN 
 - Browser: Chrome, Firefox, Edge 

**Your config.json file**
```
{
	"$schema": "https://raw.githubusercontent.com/Ylianst/MeshCentral/master/meshcentral-config-schema.json",
	"__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the name. Refer to the user's guide for details.",
	"__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
	"settings": {
		"cert": "mycertificate.certificate",
		"WANonly": true,
		"LANonly": true,
		"sessionKey": "MyReallySecretPassword1",
		"port": 443,
		"_aliasPort": 443,
		"redirPort": 80,
		"_redirAliasPort": 80,
		"mongoDb": "mongodb://VM-Mesh2.mydomain:27020/?replicaSet=meshcentral0",
		"mongoDbChangeStream": true,
		"mongoDbBulkOperations": true,
		"mongoDbName": "MeshCentral"
	},
	"domains": {
		"": {
			"title": "MeshCentral",
			"_auth": "ldap",
			"_title2": "Le Mesh'DL",
			"minify": true,
			"newAccounts": true,
			"_certUrl": "127.0.0.1",
			"userNameIsEmail": true,
			"PasswordRequirements": {
				"min": 8,
				"max": 128,
				"upper": 1,
				"lower": 1,
				"numeric": 1,
				"nonalpha": 1
			},
			"_letsencrypt": {
				"__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
				"email": "myemail@mydomain.com",
				"names": "myserver.mydomain.com",
				"production": false
			}
		},
		"peers": {
			"serverId": "Central3",
			"servers": {
				"Central1": {
					"url": "wss://VM-Mesh2.mydomain:443/"
				},
				"Central2": {
					"url": "wss://VM-Mesh3.mydomain:443/"
				},
				"Central3": {
					"url": "wss://VM-Mesh4.mydomain:443/"
				}
			}
		}
	}
}
```

**Comment by Ylianst** [comment]
For setting up MeshCentral in peering mode where you have many MeshCentral servers that each handle some of the connections, [check out this video](https://www.youtube.com/watch?v=j8xIRGScju0). It should show you the basics of how to set it up and test that it works.

**Comment by KellyDi0te** [comment]
Thank you for the answers.
Unfortunately, here is what i have now on server2 : 

`
Peer sign fail 2
PEER: InPeer: Invalid signature.
PEER: InPeer: Soft disconnect null (172.16.7.95)    (IP of server1) 
PEER: InPeer disconnect undefined (172.16.7.95)
PEER: OutPeer central3: Connecting to: wss://VM-Mesh4.my.domain:443/meshserver.ashx
PEER: OutPeer central3: Error: Error: connect ECONNREFUSED 172.16.7.15:443    (IP of server3)
PEER: OutPeer central3: Disconnected
PEER: InPeer: Connected (172.16.7.95)
Peer sign fail 2
PEER: InPeer: Invalid signature.
PEER: InPeer: Soft disconnect null (172.16.7.95)
PEER: InPeer disconnect undefined (172.16.7.95)
`
It goes like this as long as the server is on. 

And it's quite the same on server 3, you just change the ip of server3 with server2 

Do you have any idea of how to resolve this ?

**Comment by KellyDi0te** [comment]
Thank you for the answers. 

I have now a new error that i can't resolve...
Here is what i get from server2 
`
PEER: OutPeer central3: Connecting to: wss://VM-Mesh4.hdl.pri:443/meshserver.ashx
PEER: OutPeer central3: Connected
PEER: OutPeer: Server hash mismatch.
PEER: OutPeer central3: Disconnected
PEER: InPeer: Connected (172.16.7.95)
Peer sign fail 2
PEER: InPeer: Invalid signature.
PEER: InPeer: Soft disconnect null (172.16.7.95)
PEER: InPeer disconnect undefined (172.16.7.95)
`

And from server 3 : 
`
PEER: InPeer: Connected (172.16.7.38)
Peer sign fail 2
PEER: InPeer: Invalid signature.
PEER: InPeer: Soft disconnect null (172.16.7.38)
PEER: InPeer disconnect undefined (172.16.7.38)
`

Do you have any idea of how to resolve this ? 
Thank you.

**Comment by si458** [comment]
Related #5373

**Comment by si458** [comment]
fix has been issued, a new release will fix this issue please follow #5373
